app/views/decidim/time_tracker/time_tracker/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= render partial: "decidim/shared/component_announcement" %>
+
+<div class="row">
+  <div id="time_tracker" class="columns">
+    <% if !user_signed_in? %>
+      <%= render partial: "sign_in" %>
+    <% end %>
+    <% tasks.each do |task| %>
+      <h2> <%= translated_attribute task.name %> </h2>
+        <div id="activities">
+          <% task.activities.each do |activity| %>
+            <% if activity.active %>
+              <% if Decidim::TimeTracker::Assignee.find_by(activity: activity, user: current_user).status == "accepted" %>
```

Comment:
This kind of check is better to do it with a helper in the controller, something like `assignee_accepted?`